src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/ProxyHeaderModule.Framework.cs

```diff
@@ -34,12 +34,53 @@ public void Init(HttpApplication context)
 
             context.BeginRequest += (s, e) => UseOptions(((HttpApplication)s).Context.Request);
         }
+
+        context.PreSendRequestHeaders += (s, e) => FixupLocationHeader(((HttpApplication)s).Context);
```

Comment:
I wasn't familiar with this event, but the docs have this dire warning that I think we would be included in: https://docs.microsoft.com/en-us/dotnet/api/system.web.httpapplication.presendrequestheaders?view=netframework-4.8#remarks

Reply:
Removed since this is no loner necessary.